src/__tests__/register-test.js

```diff
@@ -358,6 +358,132 @@ describe('ethrResolver', () => {
       })
 
     })
+
+    describe('revoke publicKey', () => {
+      describe('Secp256k1VerificationKey2018', () => {
+        beforeAll(async () => {
+          await registry.revokeAttribute(
+            identity,
+            stringToBytes32('did/pub/Secp256k1/veriKey'),
+            '0x02b97c30de767f084ce3080168ee293053ba33b235d7116a3263d29f1450936b71',
+            { from: owner }
+          )
+          sleep(1)
+        })
+        it('resolves document', () => {
+          return expect(resolve(did)).resolves.toEqual({
+            '@context': 'https://w3id.org/did/v1',
+            id: did,
+            publicKey: [{
+              id: `${did}#owner`,
+              type: 'Secp256k1VerificationKey2018',
+              owner: did,
+              ethereumAddress: owner
+            }, {
+              id: `${did}#delegate-1`,
+              type: 'Secp256k1VerificationKey2018',
+              owner: did,
+              ethereumAddress: delegate2
+            }, {
+              id: `${did}#delegate-3`,
```

Comment:
There is no `#delegate-2` because the event for its revocation occurred after `#delegate-3` was already added.  Is this the intended behavior or should the resolved doc list delegates in sequential order?

Reply:
It is mostly intended behavior. These id's could be used to refer to keys that are no longer valid